### DISCUSSION TITLE
[求助]为什么我的表情包插件会一股脑把表情列出？

### DISCUSSION THREAD

**Discussion by March3th** [share121/inter-knot#2182] [其他]
我安装了Dave-12138大佬的表情包插件，但它会在输入框下列出一个超长的列表，不会有滚动折叠功能。导致我每次发表都要拉很长一段距离。有没有哪位好心人帮我看看，解决一下。
![屏幕截图 2024-08-15 115525](https://github.com/user-attachments/assets/da541942-866a-4e1d-814c-d34dcce542b3)
视频：
https://github.com/user-attachments/assets/aacdaf70-cca6-439e-b9c7-c936d79e01c1

**Comment by SuiHanjun** [comment]
不知道<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >

**Comment by Psinary** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/4a102d21856350263e83278ac8c48526.png" alt="吃惊" width="75" >，样式出错了吧，挑一挑就行了

**Comment by fzdqz** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/07/25/0302be52d1922a94378bd529cefbda6e_1187257128792751951.png" alt="诺登斯_盯" width="75" >会这样吗？
